apis/core/bootstrap/shapes/column-mapping.ts

```diff
@@ -66,12 +67,77 @@ ${shape('column-mapping/create')} {
       ${sh.order} 60 ;
     ] ;
   .
-
   ${xsd.string} ${rdfs.label} "string" .
   ${xsd.int} ${rdfs.label} "int" .
   ${xsd.float} ${rdfs.label} "float" .
   ${xsd.decimal} ${rdfs.label} "decimal" .
   ${xsd.date} ${rdfs.label} "date" .
   ${xsd.dateTime} ${rdfs.label} "datetime" .
 }
+
+${shape('column-mapping/reference')} {
+  ${shape('column-mapping/reference')} a ${sh.NodeShape}, ${hydra.Resource} ;
+    ${sh.targetClass} ${cc.ReferenceColumnMapping} ;
+    ${sh.property} [
+      ${sh.name} "Target Property" ;
+      ${sh.path} ${cc.targetProperty} ;
+      ${sh.minCount} 1 ;
+      ${sh.maxCount} 1 ;
+      ${dash.editor} ${editor.PropertyEditor} ;
+      ${sh.message} "A target property must be an URI or string including only letters, numbers and hyphens" ;
+      ${sh.or} (
+        [
+          ${sh.nodeKind} ${sh.IRI} ;
+        ]
+        [
+          ${sh.nodeKind} ${sh.Literal} ;
+          ${sh.datatype} ${xsd.string} ;
+          ${sh.pattern} "^[a-zA-Z0-9]+[a-zA-Z0-9-/]*$" ;
+        ]
+      );
+      ${sh.order} 10 ;
+    ] ;
+    ${sh.property} [
+      ${sh.name} "Table" ;
+      ${sh.path} ${cc.referencedTable} ;
+      ${sh.minCount} 1 ;
+      ${sh.maxCount} 1 ;
+      ${sh.class} ${cc.Table} ;
+      ${sh.order} 20 ;
+      ${dash.editor} ${dash.InstancesSelectEditor} ;
+    ] ;
+    ${sh.property} [
+      ${sh.name} "Identifier mapping" ;
+      ${sh.path} ${cc.identifierMapping} ;
+      ${sh.node} ${$rdf.blankNode('identifier-mapping')} ;
+      ${dash.editor} ${dash.DetailsEditor} ;
+      ${sh.order} 30 ;
+    ] ;
+  .
+
+  ${$rdf.blankNode('identifier-mapping')} a ${sh.NodeShape} ;
```

Comment:
This is what stardog 6 hates. In case will make it wok after all

```suggestion
      ${sh.node} ${$rdf.blankNode('identifierMapping')} ;
      ${dash.editor} ${dash.DetailsEditor} ;
      ${sh.order} 30 ;
    ] ;
  .

  ${$rdf.blankNode('identifierMapping')} a ${sh.NodeShape} ;
```